### PR TITLE
Fixes antags not getting extra TC for showing up in the report.

### PR DESCRIPTION
**Pull request by gbasood**
Literally just moved some code around so no matter what, if you have an uplink, you get extra TC for showing up on the report. Always.
🆑 
 - bugfix: Antags who show up in the round start command report will always get their extra TC now.
 - bugfix: Antags will also be warned every time now if they show up in the report.

### CODE REVIEW

**Comment by Exxion** [comment]
Initiate powergaming protocol

**Comment by SonixApache** [comment]
title typo?
changelog?

**Comment by gbasood** [comment]
i realized that my fix gave them extra TC and warned them every time even if they didnt get added as a suspect
that is now fixed

File: code/game/gamemodes/game_mode.dm [hunk 1]
```diff
@@ -184,33 +184,38 @@
...
+			became_a_suspect = TRUE
 		// Antags
 		else if(special_role == "traitor" && prob(40) || \
 		   special_role == "Changeling" && prob(50) || \
 		   special_role == "Cultist" && prob(30) || \
 		   special_role == "Head Revolutionary" && prob(30))
 			suspects += man
+			became_a_suspect = TRUE
+		// Some poor people who were just in the wrong place at the wrong time..
+		else if(prob(10))
```
Comment (Exxion) [inline code/game/gamemodes/game_mode.dm]: Ouch, my atomicity

**Comment by Exxion** [comment]
Rather than having a whole additional variable you have to set if they get found out, why not just check `man in suspects`?
Or you could iterate through `suspects` in a separate loop and give the TC there

**Comment by PJB3005** [comment]
Honestly just remove the report entirely.

If sec actually gave a damn you'd optimally search everybody on the report. If they're antag valids secured, if they're not "sorry you're on the report".

**Comment by Shadowmech88** [comment]
But the report even mentions that you're not supposed to take any action based solely on the fact that those crewmembers are on the report, you have to have another reason first.

**Comment by PJB3005** [comment]
Do you honestly think that will stop our average power gamer.

**Comment by Exxion** [comment]
Getting bwoinked for it will.
Which I saw happen yesterday.

**Comment by Allspice** [comment]
so it's just there to trick idiots into getting secban

It'd be better to split the difference, remove the report and give all traitors 12 TC.

**Comment by gbasood** [comment]
That sort of action contradicts the feedback  given in #13631. And it'd require a server poll.
And also, this is a PR to fix a bug where traitors wouldn't get TC at all. If you want to remove it, that's an entirely separate issue.

**Comment by Probe1** [comment]
@PJB3005 It has a _tenuous_ place in the game.  When sec is really really bored it gives them someone to follow.  For people too stupid to read the report and they arrest players simply for being on it well, we have admins.

**Pull request merged**